pkg/project/registry/rest.go

```diff
@@ -0,0 +1,68 @@
+package registry
```

Comment:
Nope.  Meant to stay here.  Aligns with upstream pattern where strategies are independent of storage implementation.

Sent from my iPhone

> On Apr 6, 2015, at 4:07 PM, David Eads <notifications@github.com> wrote:
> 
> In pkg/project/registry/rest.go:
> 
> > @@ -0,0 +1,68 @@
> > +package registry
> Accidentally bare in the registry package? Looks like it should be under proxy
> 
> —
> Reply to this email directly or view it on GitHub.
>